src/screens/BackupSheet.js

```diff
@@ -230,6 +230,11 @@ export default function BackupSheet() {
     additionalTopPadding = true;
   }
 
+  if (android && step === WalletBackupStepTypes.existing_user) {
+    // on this step we have 3 lines of text instead of 2
+    sheetHeight += 26;
```

Comment:
This is measured by the size our text takes there right?

Reply:
Since we have text scaling disabled in the app, I just measured the missing heigh and added it here. That's it.
This is how the whole thing works, which is horribly wrong, of course, but it's better than nothing.